manuscript/manuscript.Rmd

```diff
@@ -459,8 +461,9 @@ we now introduce a selection of appropriate techniques to answer them.
 
 For linear models and their generalizations, and some ML models, measures of
 variable importance can be obtained from the fitted model structure. In the
-case of GLMs, one can observe the magnitudes of the $t$-statistics, whereas
-for random forests, one can use out-of-bag errors [@breimanRandomForests2001].
+case of GLMs, one might observe the magnitudes of the $t$-statistics, step-wise
```

Comment:
Let's leave out the stepwise bit, lest we're associated with condoning stepwise methods which are widely known to be flawed but are somehow still held on to by the actuarial profession ;)

(see e.g. https://stats.stackexchange.com/questions/20836/algorithms-for-automatic-model-selection/20856#20856)

Reply:
Fair. I don't love stepwise, I just don't love t-statistics either because they don't always relate to predictiveness. I guess I figured we were just giving a variety of ideas.